doc/api/util.md

```diff
@@ -233,8 +233,8 @@ corresponding argument. Supported specifiers are:
   circular references.
 * `%o` - `Object`. A string representation of an object with generic JavaScript
   object formatting. Similar to `util.inspect()` with options
-  `{ showHidden: true, showProxy: true }`. This will show the full object
-  including non-enumerable properties and proxies.
+  `{ showHidden: true, showProxy: true, depth: 4 }`. This will show the full
+  object including non-enumerable properties and proxies.
```

Comment:
Note that this is just documenting existing behaviour. `%o` is already defaulting to depth of `4` before this change here.